templates/default/systemd_unit.erb

```diff
@@ -5,19 +5,23 @@ Wants=network-online.target
 After=network-online.target
 
 [Service]
+<% if @install_type != 'tarball' -%>
 Type=notify
+<% end %>
 RuntimeDirectory=elasticsearch
 PrivateTmp=true
 Environment=ES_HOME=/usr/share/elasticsearch
 Environment=ES_PATH_CONF=/etc/elasticsearch
 Environment=PID_DIR=/var/run/elasticsearch
+<% if @install_type != 'tarball' -%>
 Environment=ES_SD_NOTIFY=true
+<% end %>
 EnvironmentFile=-<%= @default_dir %>/<%= @program_name %>
 
 WorkingDirectory=<%= @path_home %>
 
-User=elasticsearch
-Group=elasticsearch
+User=<%= @es_user %>
```

Comment:
Elasticsearch recommended these be overridden using systemd's "drop-in" file and not to customize their systemd unit, which is why the cookbook did things that way originally.

Reply:
Have you got an example of how we can do a drop-in file with Chef? The old way was slightly confusing.

Reply:
It was just a file resource with some content provided via that attribute. It's really not chef specific, but I think you're right that an example would be helpful...